### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by CWiswell-IDM**

### CODE REVIEW

**Comment by CWiswell-IDM** [comment]
@YeChen-IDM and @ChJones-IDM, let's talk about this on Monday!

File: measles_kurt/Assets/schema.json [hunk 1]
```diff
@@ -0,0 +1,12054 @@
+{
```
Comment (YeChen-IDM) [inline measles_kurt/Assets/schema.json:1]: We may not need to check in the schema file which may be downloaded from get_model_file() in the test_***.py.

Comment (CWiswell-IDM) [reply measles_kurt/Assets/schema.json]: Removed!

**Review by SBuxton-IDM — APPROVED** [review]
Tried the tests and everything works as I would expect.

**Pull request merged**